src/reporters/build_kite_cli.ts

```diff
@@ -0,0 +1,123 @@
+/**
+ * MIT License
+ *
+ * Copyright (c) 2020-present, Elastic NV
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ */
+
+import { red, green, cyan } from 'kleur/colors';
+import { ReporterOptions } from '.';
+
+import BaseReporter, { renderDuration } from './base';
+import { indent, now, symbols } from '../helpers';
+import {
+  JourneyEndResult,
+  JourneyStartResult,
+  StepEndResult,
+} from '../common_types';
+import { Journey, Step } from '../dsl';
+import { renderError } from './reporter-util';
+
+export default class BuildKiteCLIReporter extends BaseReporter {
```

Comment:
This is how I am seeing it, Are you okay with this format? If you have the output from build kite once these suggested changes are pushed, please do share it. 

<img width="695" alt="Screenshot 2023-12-06 at 3 22 03 PM" src="https://github.com/elastic/synthetics/assets/3902525/2db8bb7b-b936-4a30-b250-baad6d6068df">

Reply:
yes, that looks good to me